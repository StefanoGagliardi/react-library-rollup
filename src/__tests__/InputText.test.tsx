/* eslint-disable @typescript-eslint/no-var-requires */
import '@testing-library/jest-dom/extend-expect';

import { configure } from 'enzyme';
const Adapter = require('enzyme-adapter-react-16');
configure({ adapter: new Adapter() });
import React from 'react';
import { shallow } from 'enzyme';

describe('Hi, first Typescript unit test', () => {
  it('First simple unit test', () => {
    const a = 5;
    expect(a).toEqual(5);
  });

  it('renders', () => {
    const wrapper = shallow(
      <div>
        <h1>Hello, Enzyme!</h1>
      </div>,
    );
    expect(wrapper.find('h1').html()).toMatch(/Hello, Enzyme/);
  });

  it('renders snapshots, too', () => {
    const wrapper = shallow(
      <div>
        <h1>Hello, Enzyme!</h1>
      </div>,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
