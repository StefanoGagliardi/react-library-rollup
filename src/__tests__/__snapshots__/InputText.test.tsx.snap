// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hi, first Typescript unit test renders snapshots, too 1`] = `ShallowWrapper {}`;
